cmake/DependenciesConfig.cmake

```diff
@@ -6,7 +6,7 @@ add_external_lib(
         sdl/2.0.20
         REQUIRED
         INTERFACE_NAME SDL::SDL
-        PKG_CONFIG "sdl2 >= 2.0"
+        PKG_CONFIG "sdl2 >= 2.0.17"

```

Comment:
this caused the linux build to fail

Reply:
This need, because imgui need sdl 2.0.17 and better for work

Reply:
That’s not what cause the build to fail, but it doesn’t work either.
We need a newer version of sdl and the Linux repos are infamous for providing really old libraries and stuff

Reply:
@SimplyLiz 
![image](https://user-images.githubusercontent.com/12609468/171117819-9d02cf05-b874-4cbf-a86c-4c1e04d7bd3e.png)
according to the logs it is or I'm understanding this the wrong way 

what caused the fail then ?

Reply:
it was added for special, because imgui need better than 2.0.14